plugins/europeana/entity.js

```diff
@@ -296,21 +296,22 @@ function getRelatedEntityTitleLink(entities) {
  * If type is topic, use note
  * If type is person, use biographicalInformation
  * @param {Object} entity data
+ * @param {string} locale Locale code for desired language
  * @return {String} description when available in English
  * TODO: l10n
  */
-export function getEntityDescription(entity) {
+export function getEntityDescription(entity, locale) {
   if (!entity) return null;
   let description;
   if (entity.type === 'Concept' && entity.note) {
-    description = entity.note.en ? entity.note.en[0] : '';
+    description = langMapValueForLocale(entity.note, locale);
   } else if (entity.type === 'Agent' && entity.biographicalInformation) {
     // check if biographicalInformation is an array of objects
     // TODO: it _should_ always be an array. this is an Entity API bug. remove
     //       the condition when fixed upstream.
     //       see: https://europeana.atlassian.net/browse/EA-1685
     if (entity.biographicalInformation.length !== undefined) {
-      description = entity.biographicalInformation.filter(info => info['@language'] === 'en')[0]['@value'];
+      description = langMapValueForLocale(entity.biographicalInformation, locale);
```

Comment:
This does not appear to correctly handle the malformed case where biographicalInformation is not an array, e.g. /en/entity/person/37430-antonio-monticini